src/network/serverpackethandler.cpp

```diff
@@ -794,7 +794,7 @@ void Server::process_PlayerPos(RemotePlayer *player, PlayerSAO *playersao,
 	u32 keyPressed = 0;
 
 	// default behavior (in case an old client doesn't send these)
-	f32 fov = (72.0*M_PI/180) * 4./3.;
```

Comment:
The default is not needed anymore, clientiface does the right thing. Accidentally came back when I reverted back to sending the single player pos packet only.